### ISSUE TITLE
How it works?

### ISSUE THREAD

**Issue by gwerbin** [AppliedDataSciencePartners/xgboostExplainer#13]
Thanks for open-sourcing this package, and for providing the nice writeup at https://medium.com/applied-data-science/new-r-package-the-xgboost-explainer-51dd7d1aa211

Would it be possible to publish a short explanation of the algorithm this uses? Even just bullet points in the readme would be really helpful. It would make it much easier for someone else to port this to Python, or to other gradient boosting methods like LightGBM.

**Comment by steventsimpson** [comment]
here's a python port : https://github.com/gameofdimension/xgboost_explainer
